<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="3" total="6" passed="2" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-05-04T18:57:42 IST" name="Default suite" finished-at="2023-05-04T18:57:59 IST" duration-ms="17278">
    <groups>
    </groups>
    <test started-at="2023-05-04T18:57:42 IST" name="Default test" finished-at="2023-05-04T18:57:59 IST" duration-ms="17278">
      <class name="com.blinkit.test.TrainBookingTest">
        <test-method is-config="true" signature="SteUp()[pri:0, instance:com.blinkit.test.TrainBookingTest@cb0755b]" started-at="2023-05-04T18:57:42 IST" name="SteUp" finished-at="2023-05-04T18:57:50 IST" duration-ms="7818" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SteUp -->
        <test-method signature="pageTitleTest()[pri:0, instance:com.blinkit.test.TrainBookingTest@cb0755b]" started-at="2023-05-04T18:57:50 IST" name="pageTitleTest" finished-at="2023-05-04T18:57:51 IST" duration-ms="1368" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- pageTitleTest -->
        <test-method signature="SearchResult_Test()[pri:0, instance:com.blinkit.test.TrainBookingTest@cb0755b]" started-at="2023-05-04T18:57:51 IST" name="SearchResult_Test" finished-at="2023-05-04T18:57:57 IST" depends-on-methods="com.blinkit.test.TrainBookingTest.pageTitleTest" duration-ms="5974" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SearchResult_Test -->
        <test-method signature="BookBTn_Test()[pri:0, instance:com.blinkit.test.TrainBookingTest@cb0755b]" started-at="2023-05-04T18:57:57 IST" name="BookBTn_Test" finished-at="2023-05-04T18:57:59 IST" depends-on-methods="com.blinkit.test.TrainBookingTest.SearchResult_Test" duration-ms="2094" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:819)
at org.testng.Assert.assertEquals(Assert.java:829)
at com.blinkit.test.TrainBookingTest.BookBTn_Test(TrainBookingTest.java:60)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- BookBTn_Test -->
      </class> <!-- com.blinkit.test.TrainBookingTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
